osbs/api.py

```diff
@@ -713,6 +713,46 @@ def get_build_logs(self, build_id, follow=False, build_json=None, wait_if_missin
 
         return logs
 
+    def _parse_build_log_entry(self, entry):
+        items = entry.split()
+        if len(items) < 4:
+            logger.warning("This is not a valid build log entry: %s", entry)
+            return (None, entry)
+        date = items[0]
+        time = items[1]
+        platform = items[2]
+        plen = len(date) + len(time) + len(platform) + 3
```

Comment:
Yeah, I'll go with that, it does make it much clearer what that constant is there for.